packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -226,10 +197,6 @@ export default class Calendar extends React.Component {
      * Prop to show short names of weekdays .
      */
     useWeekdaysShort: PropTypes.bool,
-    /**
-     * Prop to open a separate portal version of date picker .
-     */
-    withPortal: PropTypes.bool,
```

Comment:
Don't we need this for date-picker...?

Reply:
Yes, we do but it is not needed in `Calendar.jsx` because it is being passed down _only_ to the `Time` component which will be irrelevant with this change.